replicaset.go

```diff
@@ -0,0 +1,88 @@
+package vshard_router
+
+import (
+	"context"
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/pool"
+)
+
+type ReplicasetInfo struct {
+	Name string
+	UUID uuid.UUID
+}
+
+type Replicaset struct {
+	conn *pool.ConnectionPool
+	info ReplicasetInfo
+
+	bucketCount atomic.Int32
+}
+
+type ReplicasetCallOpts struct {
+	PoolMode pool.Mode
+	Timeout  time.Duration
+}
+
+// ReplicasetCallImpl perform function on remote storage
+func (rs *Replicaset) ReplicasetCallImpl(
+	ctx context.Context,
+	opts ReplicasetCallOpts,
+	fnc string,
+	args interface{},
+) (interface{}, StorageResultTypedFunc, error) {
+	if opts.Timeout == 0 {
+		opts.Timeout = CallTimeoutMin
+	}
+
+	timeout := opts.Timeout
+	timeStart := time.Now()
+
+	req := tarantool.NewCallRequest(fnc)
```

Comment:
If u call some function directly without vshard.storage.call, u cannot get some errors if ur bucket was moved to other replicaset

Reply:
replicaset's call there is no need to use vshard.storage.call, because
1) user want to call function without pass bucketID
2) from source_code https://github.com/tarantool/vshard/blob/master/vshard/replicaset.lua#L661 - router not processing error with bucket's moving

Reply:
it triggered me cause origin function name and ur name are different

Reply:
agreed, so name "replica_call" will be fit